Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11
+
+WORKDIR /cinema
+
+COPY requirements.txt .
```

Comment:
It's a good practice to copy only the necessary files for installing dependencies. Consider using `COPY requirements.txt /app/` and then setting `WORKDIR /app` before running the pip install command.